examples/triton-docker-combined/triton.tf

```diff
@@ -0,0 +1,48 @@
+# See http://www.joyent.com/blog/introducing-hashicorp-terraform-for-joyent-triton
+
+provider "triton" {
+  account = "${var.triton_account}"
+  key = "${var.triton_key_path}"
+  key_id = "${var.triton_key_id}"
+  url = "${var.triton_url}"
+}
+
+provider "docker" {
+  host = "${var.docker_host}"
+  cert_path = "${var.docker_cert_path}"
+}
+
+resource "docker_image" "nginx" {
+  name = "nginx:latest"
+  keep_updated = true
+}
+
+resource "docker_container" "nginx" {
+  count = 1
+  name = "nginx-terraform-${format("%02d", count.index+1)}"
+  image = "${docker_image.nginx.latest}"
+  must_run = true
+
+  env = ["env=test", "role=test"]
+
+  ports {
+    internal = 80
+    external = 80
+  }
+}
+
+resource "triton_machine" "testmachine" {
+  name = "test-machine"
+  package = "t4-standard-512M"
+  image = "ffe82a0a-83d2-11e5-b5ac-f3e14f42f12d"
+
+  count = 1
+}
+
+resource "triton_machine" "windowsmachine" {
+  name = "win-test-terraform"
+  package = "333814c2-b4a7-481f-9f86-73bb4122c7a3"
```

Comment:
One more reason to switch to names is that the UUIDs change. Our demo doesn't work in SW-1 today because this morning the UUID of the package I was using for the Windows example changed.